### PR TITLE
Assorted fixes so that validation passes

### PR DESCRIPTION
**Pull request by apdavison**
This is the first pull request with basic tests in place, and contains a few small fixes to make sure that validation passes, at least for the single dataset that is currently tested.

### CODE REVIEW

File: bids2openminds/main.py [hunk 1]
```diff
@@ -132,7 +132,10 @@ def subjects_creation(subject_id, layout_df, layout):
         state_cash = []
         for sesion in sessions:
```
Comment (lzehl) [inline bids2openminds/main.py:133]: can we fix the typo here `for session in sessions` ?

File: bids2openminds/main.py [hunk 2]
```diff
@@ -19,17 +19,17 @@ def create_techniques(layout_df):
...
     for datatype in datatypes:
         if not (pd.isna(datatype)):
-            approaches.extend(bids2openminds_instance(datatype, "MAP_2_EXPERIMENTAL_APPROACHES"))
+            approaches.update(bids2openminds_instance(datatype, "MAP_2_EXPERIMENTAL_APPROACHES"))
 
-    return approaches
+    return list(approaches) or None
 
 
 def dataset_version_create(bids_layout, dataset_description, layout_df, studied_specimens, file_repository):
```
Comment (lzehl) [inline bids2openminds/main.py:35]: could we change this in this PR already to `def create_dataset_version(...)` for consistency (cf. `def create_approaches(...)`)

Comment (lzehl) [reply bids2openminds/main.py:35]: leave it to the next PR if too much refactoring is required

File: bids2openminds/main.py (hunk 1 above)
Comment (lzehl) [reply bids2openminds/main.py:133]: leave it to the next PR if too much refactoring is required

Comment (Peyman-N) [reply bids2openminds/main.py:133]: It should be a fairly simple PR, but I think we should do it in another pull request, as it is not related to this one.

File: bids2openminds/main.py (hunk 2 above)
Comment (Peyman-N) [reply bids2openminds/main.py:35]: Agreed but again I think we should do it in another pull request.

**Review by Peyman-N — APPROVED** [review]

**Pull request merged**